pipeline-scripts/commonlib.groovy

```diff
@@ -85,6 +85,24 @@ def cleanCommaList(str) {
     return str.replaceAll(',', ' ').split().join(',')
 }
 
+def retry(tries=3, Closure it) {
```

Comment:
Are you using this custom `retry` function instead of the stdlib `retry` because of anything specific (the closure?)

Reply:
@tbielawa the stdlib retry has a super annoying syntax IMHO... don't really blame him.

Reply:
No, I just wasn't aware it existed. Should have suspected :) All mine adds is a bit of output that's not too useful. I'll remove it.

Reply:
Updated...